### DISCUSSION TITLE
[v5] Do transactions are supported in v5?

### DISCUSSION THREAD

**Discussion by naveen1941** [adonisjs/core#1740] [Help]
Do transactions are supported in v5? 
As per this https://trello.com/c/8lPTtuWl/13-support-for-translations it's still in progress. But [here](https://preview.adonisjs.com/guides/database/transactions) the documentation about v5 transaction is present. So, does the transactions are supported in v5?
I am thinking to consider adonis for creating an application which deals with issuing charge, refund requests to payment gateway. At this point can I use v5 for my application which is a transaction sensitive?

**Comment by akraminakib** [comment]
@naveen1941 The trello card you're referring to is for translations (internationalization), not transactions. Transactions is already supported in V5 as per the documentation you linked.

**Comment by FeMaffezzolli** [comment]
@naveen1941  Are you talking about database transactions or translations?
